packages/core/src/lib/resource-pill.svelte

```diff
@@ -0,0 +1,69 @@
+<!--
+@component
+  
+For displaying a list of items.
+
+```svelte
+<ResourcePill value="BeagleBone" on:remove={onRemove} />
+```
+-->
+<svelte:options immutable />
+
+<script lang="ts">
+import { Icon, type IconName } from '$lib';
+import cx from 'classnames';
+
+import { createEventDispatcher } from 'svelte';
+
+/** The text in the resource pill. */
+export let value = '';
+
+/** Whether or not the resource pill is removable. If removable, there is a button to remove the resource pill. */
+export let removable = false;
+
+/** Additional CSS classes to pass to the resource pill. */
+let extraClasses: cx.Argument = '';
+export { extraClasses as cx };
+
+/** The icon shown in the button. */
+export let icon: IconName;
+
+const dispatch = createEventDispatcher<{
+  /** When the resource pill is removed. */
+  remove: { value: string };
```

Comment:
If we switch to `<slot />`, I think we should remove the `value` from the event. The parent can distinguish via how it attached `on:remove` listeners, and I think it complicates the component to include it